scripts.js

```diff
@@ -0,0 +1,226 @@
+class Quiz {
+  constructor(properties) {
+    this.preloader = properties.preloader;
+    this.correctRequestAnswers = properties.correctRequestAnswers;
+    this.allAnswersRequest = properties.allAnswersRequest;
+    this.correctDefaultAnswers = properties.correctDefaultAnswers;
+    this.questionRequest = properties.questionRequest;
+    this.requestLink = properties.requestLink;
+    this.checkResultBind = this.checkResult.bind(this);
+    this.start = properties.start;
+    this.next = properties.next;
+    this.previous = properties.previous;
+    this.slides = properties.slides;
+    this.restart = properties.restart;
+    this.container = properties.container;
+    this.form = properties.form;
+    this.formName = properties.formName;
+    this.buttonName = properties.buttonName;
+    this.counterAnswers = properties.counterAnswers;
+    this.regex = properties.regex;
+    this.countSlide = 0;
+    this.countAnswers = 0;
+    this.timerShowQuestion;
+    this.timerCheckQuestion;
+  }
+
+  makeRequest() {
+    fetch(this.requestLink)
+      .then(res => res.json())
+      .then(json => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/voronkova-ab/game-js/pullRequest?prid=7705696)
```suggestion
      .then((json) => {
```